plugins/inputs/sqlserver/sqlserverqueries.go

```diff
@@ -404,6 +404,7 @@ SELECT DISTINCT
 			,'Backup/Restore Throughput/sec'
 			,'Total Server Memory (KB)'
 			,'Target Server Memory (KB)'
+			,'Stolen target memory ratio'
```

Comment:
The exact value is:
'Stolen Server Memory (KB)'

Reply:
ah thank you!